src/core/tests/runtime/interpreter/unit_test.manifest

```diff
@@ -140,3 +140,6 @@ INTERPRETER.onnx_dyn_shapes_flatten_neg_axis
 INTERPRETER.onnx_dyn_shapes_slice_10_slice_2d_default_steps_dyn_begin_end
 INTERPRETER.onnx_model_instance_normalization_dyn_shape
 INTERPRETER.onnx_controlflow_loop_2d_no_identity_termination_cond_false
+
+# Segmentation fault (core dumped)
+INTERPRETER.onnx_duplicated_output_names
```

Comment:
This line seems to be a root cause of the segfault with INTERPRETER, with CPU the test passed:
https://github.com/openvinotoolkit/openvino/blob/6f60a7d8f098c7cf3195ba96d964a11bf1babe55/docs/template_plugin/backend/int_executable.cpp#L122